### ISSUE TITLE
security/zoneminder 1.28.1 failed to build

### ISSUE THREAD

**Issue by nirv199** [TritonDataCenter/pkgsrc-legacy#311]
This package failed to build: http://us-east.manta.joyent.com/pkgsrc/public/reports/2015Q3/x86_64/20151204.1911/zoneminder-1.28.1nb4/depends.log
Not sure if this is what conflicted:
pkg_add: Installed package `percona-cluster-5.6.21nb3' conflicts with`{mysql,percona}_-{client,server}-[0-9]_' when trying to install `mysql-client-5.6.26'.

**Comment by mamash** [comment]
I'll fix this particular problem, but ZoneMinder has hard platform checks and doesn't support Solaris officially. Here's a relevant Solaris support ticket...

https://github.com/ZoneMinder/ZoneMinder/issues/905

**Comment by nirv199** [comment]
Thank you so much Filip!

**Comment by nirv199** [comment]
sorry to bug you mamsh, just wondering if you had a chance to have a look at this? I know the netbsd ports have  been updated to latest 1.29 but i didn't see the zoneminder package on pkgin on latest 16.1 container image

**Comment by mamash** [comment]
No, the configure still fails because they hard check platforms they support, see:

http://us-east.manta.joyent.com/pkgsrc/public/reports/2016Q1/x86_64/20160524.1555/zoneminder-1.28.1nb6/configure.log

And the upstream Solaris support ticket is still open, so unlikely any progress happened on this front.

**Comment by mamash** [comment]
I'll see if anything changed in 1.29 that could help us.

**Comment by mamash** [comment]
Version 1.29 mentions SunOS in the changelog and doesn't reject SunOS rightaway, but doesn't build at all. I'll keep poking.
